packages/core-data/src/fetch/test/__experimental-fetch-link-suggestions.js

```diff
@@ -70,6 +70,18 @@ jest.mock( '@wordpress/api-fetch', () =>
 						subtype: 'page',
 					},
 				] );
+			case '/wp/v2/media?search=&per_page=20':
+				return Promise.resolve( [
+					{
+						id: 54,
+						title: {
+							rendered: 'Media',
```

Comment:
Might be worth making the title less generic, just in case a bug makes the API return the word "Media" for all items?

Reply:
Or maybe that's what it does?!

Reply:
I think you made a good point. I made the title more generic.

Reply:
Fixed.